tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
     "compilerOptions": {
+        "skipLibCheck": true,
+        "allowJs": true,
```

Comment:
Why do we need these changes?

Reply:
I am using `allowjs` to import `dialog_widget` in `upload_widget.ts` since it is a JavaScript module, and `skiplibcheck` to suppress errors that arise due to using `allowjs`. However, once dialog_widget is converted to TypeScript, these options will no longer be necessary. I have requested a solution on [CZO](https://chat.zulip.org/#narrow/stream/21-provision-help/topic/use.20javascript.20module.20in.20typescript/near/1533464) and will update the PR once I receive one.